cmd/prometheus/main.go

```diff
@@ -204,7 +204,7 @@ func main() {
 	a.Flag("storage.tsdb.retention.time", "How long to retain samples in storage. When this flag is set it overrides \"storage.tsdb.retention\". If neither this flag nor \"storage.tsdb.retention\" nor \"storage.tsdb.retention.size\" is set, the retention time defaults to "+defaultRetentionString+". Units Supported: y, w, d, h, m, s, ms.").
 		SetValue(&newFlagRetentionDuration)
 
-	a.Flag("storage.tsdb.retention.size", "[EXPERIMENTAL] Maximum number of bytes that can be stored for blocks. Units supported: KB, MB, GB, TB, PB. This flag is experimental and can be changed in future releases.").
+	a.Flag("storage.tsdb.retention.size", "[EXPERIMENTAL] Maximum number of bytes that can be stored for blocks. Make sure to specify the unit suffix. Ex: \"512MB\". Units supported: KB, MB, GB, TB, PB. This flag is experimental and can be changed in future releases.").
```

Comment:
B is also supported.

The new text doesn't feel right. Maybe something like A unit is required, supported units:

Reply:
Be aware that a unit is required, supported units: B, KB, MB, GB, TB, PB. Ex: "512MB"

Is that ok?